README.md

```diff
@@ -1,47 +1,51 @@
+All pages on the site correspond to a Markdown file (.md) which can be found inside `pages/Documents`. To update a file, please branch off of the `master` branch, edit the file in question, and create a pull request back to the master branch.
 
-#### Committing Changes to the Developers Site
+### Committing changes to the site
 
-Before making any commits, please make sure to read the Updating/Creating Headers section. There is now a Git precommit hook that makes sure you follow the rules on Markdown file creation. This hook will run on every commit and deny commits if they fail the test. The errors will be outputed to `./_scripts/docOutputError.log`. If you are adding new content, please make sure you are updating the content in documentsupdated.yaml file. Our tests will use the documents updated yaml file as the source of truth. So make sure your header naming structure matches the documentsupdated.yaml.
+Before making any commits, please make sure to read the _Updating and creating headers_ section. There is now a Git precommit hook that makes sure you follow the rules on Markdown file creation. This hook will run on every commit and deny commits if they fail the test. The errors will be outputed to `./_scripts/docOutputError.log`. If you are adding new content, please make sure you are updating the content in documentsupdated.yaml file. Our tests will use that YAML file as the source of truth. So make sure your header naming structure matches the documentsupdated.yaml.
 
 File name rules:
+
 1. All Markdown files must match the pagename that is provided in the headers
 2. Spaces in pagename must be replaced with a “-”
 3. All words are lowercased
 4. All special characters excluding periods and dashes need to be removed from the filename
-5. Periods are replaced with dashes.
+5. Periods are replaced by dashes.
 
 Folder name rules:
-1. They must be TitleCase
+
+1. They must use [upper camel case (Pascal case)](https://en.wikipedia.org/wiki/Camel_case)
 2. All files in the folder, must include a reference in its header to its folder name. This will be in either the documentname, categoryname, or subfoldername (depending on the location of the folder)
 3. All special characters excluding periods, dashes, and “&” need to be removed from Foldername
 4. Periods are replaced with dashes.
 
-E.G pagename : `Customizing the Conversational Cloud!` should be filename `customizing-the-conversational-cloud.md`
-E.G documentname: `Add Agent Widgets` should be a folder with name `AddAgentWidgets`
+Examples:
 
-* Category name will always be the top most folder in the sidebar
+* A page name `Customizing the Conversational Cloud!` should use the file name `customizing-the-conversational-cloud.md`
+* A document name `Add Agent Widgets` should use a folder with the name of `AddAgentWidgets`
 
-#### How to Understand the documentsupdated yaml File
+The category name makes for the top most folder in the sidebar.
```

Comment:
What are you trying to convey here with "makes for"?

Reply:
You mean, keep the old wording?